packages/zone/src/durable.js

```diff
@@ -54,7 +47,7 @@ const attachDurableStores = getBaggage => {
     provideDurableWeakMapStore(getBaggage(), label, options);
 
   /** @type {import('.').Stores} */
-  return Far('durableStores', {
+  return harden({
```

Comment:
I am not actually suggesting any of the changes of stores and zones from `Far` to `harden`. Rather, I did it in this PR just to make them non-passable, so I could test that I fixed the problems of where they sometimes try to store themselves.

At https://github.com/Agoric/agoric-sdk/pull/7972 I took steps to make stores and zones into self-storing exos. This failed for reasons I'll explain there. Given that failure, I think that we should still treat stores and zones as non-self storing.

So all these changes `Far` to `harden` changes: not suggested.